cruise.umple/test/cruise/umple/compiler/487_parallelStateMachines_sameNsmDiffNames.ump

```diff
@@ -0,0 +1,15 @@
+class X {
```

Comment:
In my tests  --> assertNoWarnings("487_parallelStateMachines_sameNsmDiffNames.ump");